src/homeworks/test_46/index.ts

```diff
@@ -0,0 +1,7 @@
+type dataInput = Record<"name" | "surname", string>
```

Comment:
I tipi dovrebbero cominciare con la lettera maiuscola.
```suggestion
type DataInput = Record<"name" | "surname", string>
```